packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -233,135 +234,6 @@ describe('backupNg', () => {
         status: 'failure',
         result: expect.any(Object),
       })
-
-      expect(noSuchObject.is(vmTask.result)).toBe(true)
-    })
```

Comment:
it's related to the above test, right?

Reply:
Yes, the error is a `noSushObject` anymore.

Reply:
Why is that?

What's the new result?

Which behavior is the best, the new or the old one?

Reply:
1. The new implementation changes this behavior
2. The new error message is 'no XAPI associated to non-existent-id' but the [`noSushObject` is logged and ignored](https://github.com/vatesfr/xen-orchestra/commit/e744d90dbb5800a17cb5062aa7878414ce87e9aa).
3. The most important thing is to have a comprehensible error message with a good stack, and the two implementations respect this rule

Reply:
What's the new result?

Reply:
Please see 2.